Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py

```diff
@@ -178,16 +178,18 @@ def fetch_indicators_command(client: Client, indicator_type: str, feedTags: list
         use_prefix_flat = bool(feed_config.get('flat_json_with_prefix', False))
         mapping_function = feed_config.get('mapping_function', indicator_mapping)
         handle_indicator_function = feed_config.get('handle_indicator_function', handle_indicator)
+        create_relationships_function = feed_config.get('create_relations_function')
```

Comment:
we want to have a relationship function and that the `Create relationships` param will be set to true

Reply:
That is what we talked about yesterday, that if we require the function in order to make relationships (and not providing one as a default) then it can be the parameter as well. If no function was passed in configuration, we will not make the relationships. Do you want me to add another param anyway?

Reply:
I believe so. let's say that I have this feed, but do not want to create relationships. can I avoid that without a param?